### PR TITLE
Feature/tuttorial screen

### PR DESCRIPTION
**Pull request by DenysMelnykov**

### CODE REVIEW

File: ExchangeRates/ExchangeRates/UserStories/Tutorial/CreateTutorialView.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        addSubview(descriptionLabel)
+        self.descriptionLabel.text = descriptionLabelText
+        self.descriptionLabel.textAlignment = .center
+        self.descriptionLabel.textColor = R.color.lightGrey()
+        self.descriptionLabel.numberOfLines = 0
+        self.descriptionLabel.font = R.font.helveticaNeue(size: 16)
+        descriptionLabel.snp.makeConstraints { (make) in
+            make.top.equalTo(titleLabel.snp.bottom).offset(30)
+            make.centerX.equalTo(titleLabel)
+            make.width.equalTo(titleLabel)
```
Comment (DergunovVitaly) [inline ExchangeRates/ExchangeRates/UserStories/Tutorial/CreateTutorialView.swift]: can be one line

File: ExchangeRates/ExchangeRates/UserStories/Tutorial/CreateTutorialView.swift [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            make.centerX.equalTo(imageView)
+            make.width.equalTo(225)
+            make.height.equalTo(30)
+        }
+        
+        addSubview(descriptionLabel)
+        self.descriptionLabel.text = descriptionLabelText
+        self.descriptionLabel.textAlignment = .center
+        self.descriptionLabel.textColor = R.color.lightGrey()
+        self.descriptionLabel.numberOfLines = 0
```
Comment (DergunovVitaly) [inline ExchangeRates/ExchangeRates/UserStories/Tutorial/CreateTutorialView.swift]: why we add this line:
self.descriptionLabel.numberOfLines = 0. ?
it's question

File: ExchangeRates/ExchangeRates/UserStories/Tutorial/CreateTutorialView.swift [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            make.centerX.equalTo(titleLabel)
+            make.width.equalTo(titleLabel)
+            make.height.greaterThanOrEqualTo(65)
+        }
+        
+        func coinsPosition(position: Int) {
+            descriptionLabel.snp.makeConstraints { (make) in
+                make.top.equalToSuperview().offset(position)
+            }
+        }
```
Comment (DergunovVitaly) [inline ExchangeRates/ExchangeRates/UserStories/Tutorial/CreateTutorialView.swift]: maybe this func never called? delete ?

File: ExchangeRates/ExchangeRates/UserStories/Tutorial/TutorialVC.swift [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                                                        titleLabelText: Localizable.titleTutorialThirdSlide(),
+                                                        descriptionLabelText: Localizable.descriptionTutorialThirdSlide())]
+    
+    override func loadView() {
+        let contentView = TutorialView(frame: .zero, imageArray: tutorialViewArray)
+        contentView.delegate = self
+        view = contentView
+    }
+    
+    func tupButton() {
```
Comment (DergunovVitaly) [inline ExchangeRates/ExchangeRates/UserStories/Tutorial/TutorialVC.swift]: rename pls to startButtonAction

File: ExchangeRates/ExchangeRates/UserStories/Tutorial/TutorialView.swift [hunk 5]
```diff
@@ -6,22 +6,109 @@
 //  Copyright © 2019 Denis Melnikov. All rights reserved.
 //
 
+import Foundation
 import UIKit
 
-class TutorialView: UIView {
+protocol TutorialViewDelegate: class {
+    func tupButton()
```
Comment (DergunovVitaly) [inline ExchangeRates/ExchangeRates/UserStories/Tutorial/TutorialView.swift]: startButtonAction

File: ExchangeRates/ExchangeRates/UserStories/Tutorial/TutorialView.swift [hunk 6]
```diff
@@ -6,22 +6,109 @@
...
     }
     
+    weak var delegate: TutorialViewDelegate?
+    
     private func setupLayout() {
         
+        addSubview(coinsImageView)
+        coinsImageView.contentMode = .scaleAspectFill
+        coinsImageView.image = R.image.coins()
+        coinsImageView.alpha = 0.4
```
Comment (DergunovVitaly) [inline ExchangeRates/ExchangeRates/UserStories/Tutorial/TutorialView.swift]: try to delete this line, and see what happens

Comment (DenysMelnykov) [reply ExchangeRates/ExchangeRates/UserStories/Tutorial/TutorialView.swift]: if delete these lines - coinsImageView will disappear, but the project d`nt crash

**Pull request merged**